sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/Optimizer.scala

```diff
@@ -164,10 +164,20 @@ abstract class Optimizer(sessionCatalog: SessionCatalog)
    * Optimize all the subqueries inside expression.
    */
   object OptimizeSubqueries extends Rule[LogicalPlan] {
+    private def removeTopLevelSorts(plan: LogicalPlan): LogicalPlan = {
```

Comment:
nit: `removeTopLevelSort`? (I think this func removes a single sort on the top?)